compat-28.el

```diff
@@ -392,6 +392,39 @@ not a list, return a one-element list containing OBJECT."
       object
     (list object)))
 
+(compat-defun get-buffer-create (name &optional inhibit-buffer-hooks)
+  "Return the buffer specified by BUFFER-OR-NAME, creating a new one if needed.
+If BUFFER-OR-NAME is a string and a live buffer with that name exists,
+return that buffer.  If no such buffer exists, create a new buffer with
+that name and return it.
+
+If BUFFER-OR-NAME starts with a space, the new buffer does not keep undo
+information.  If optional argument INHIBIT-BUFFER-HOOKS is non-nil, the
+new buffer does not run the hooks `kill-buffer-hook',
+`kill-buffer-query-functions', and `buffer-list-update-hook'.  This
+avoids slowing down internal or temporary buffers that are never
+presented to users or passed on to other applications.
+
+If BUFFER-OR-NAME is a buffer instead of a string, return it as given,
+even if it is dead.  The return value is never nil."
+  (if inhibit-buffer-hooks
+      (let (kill-buffer-hook
+            kill-buffer-query-functions
+            buffer-list-update-hook)
+        (get-buffer-create name))
+    (get-buffer-create name inhibit-buffer-hooks)))
```

Comment:
The argument inhibit-buffer-hooks should be removed here.

Reply:
Woops!  Thank you.